hard/tsmc_28/bsg_clk_gen/README.verification

```diff
@@ -0,0 +1,24 @@
+This clock generator is intended to be able to change the frequency as the chip operates without glitches or short pulses.
```

Comment:
Seems like these two files are replicated from 40 and 14; perhaps we should refactor them up to bsg_clk_gen; and rename the conflicting oscillator file with _nonsynth?

Reply:
Interesting idea. The bsg_clk_gen/bsg_clk_gen_osc.v is used in bsg_clk_gen/bsg_clk_gen.v which is used by both synth and nonsynth. So the portability layer is bsg_clk_gen_osc itself.

Maybe we can move it to hard/common/bsg_clk_gen/ ? For hardened modules which are "generic" between proc nodes?

Reply:
or are you suggesting bsg_nonsynth_clk_gen_osc.v would contain "bsg_clk_gen_osc" (not named nonsynth)?